.github/workflows/build.yml

```diff
@@ -67,3 +77,24 @@ jobs:
         with:
           name: lifecycle-windows-x86-64
           path: out/lifecycle-v*+windows.x86-64.tgz
+      - name: Publish lifecycle image
+        run: |
+          If ("${{ github.event_name }}" -Ne 'push') { exit }
+          echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
+          $LIFECYCLE_IMAGE_TAG=$(git describe --always --dirty)
+          $LIFECYCLE_VERSION=$(go run ./tools/version/main.go)
+          go run ./tools/image/main.go -lifecyclePath ./out/lifecycle-v${LIFECYCLE_VERSION}+windows.x86-64.tgz -tag buildpacksio/lifecycle:${LIFECYCLE_IMAGE_TAG}-windows
```

Comment:
If we didn't want to maintain both a linux and windows version of this logic we could move all image creation to the `create-lifecycle-image-manifest-list`. Windows image creation doesn't actually need to happen in a windows container. We could invoke `./tools/image/main.go` twice, once with the `-os` flag and generate both images in the linux environment.

This has the added benefit that no images will be published if either linux or windows tests fail.